bin/spaid_create_function_file

```diff
@@ -0,0 +1,39 @@
+#!/bin/zsh
+
+if [[ "$1" == "-h" ]] ; then
+    echo "Usage: `basename $0` [-h]"
+    echo 
+    echo "Run this script to create a new function file with the corresponding"
+    echo "test file."
+    echo 
+    echo "Example:"
+    echo 
+    echo "    $ spaid_create_function_file seedcase_sprout core properties"
+    echo "    Created:"
+    echo "    ./seedcase_sprout/core/properties.py"
+    echo "    ./tests/core/test_properties.py"
+    echo 
+    echo "Positional arguments:"
+    echo 
+    echo "- module: The folder of the module you want to create the new function in."
+    echo "- submodule: The folder of the submodule or part, e.g. core vs cli, that"
+    echo "  you want to create the new function in."
+    echo "- name: The name of the function you want to create."
+    exit 0
+fi
+
+pyproject_file=$(pwd)/pyproject.toml
+
+if [[ ! -f "$pyproject_file" ]]
+then
+    echo "Error: No '$pyproject_file' found. You need to be in a Python project."
+    exit 1
+fi
+
+py_file=./$1/$2/$3.py
+test_file=./tests/$2/test_$3.py
+touch py_file test_file
```

Comment:
```suggestion
touch $py_file $test_file
```
This should fix it?

Reply:
Yep, that fixes it for me 👍

Reply:
So weird, I thought I fixed that... :shrug: